build/mock-rbac/Dockerfile

```diff
@@ -0,0 +1,34 @@
+##
+# This build the container image for the service
+##
+FROM registry.access.redhat.com/ubi9/go-toolset:1.20 as builder
+LABEL idmsvc-backend=builder
+# https://developers.redhat.com/articles/2022/05/31/your-go-application-fips-compliant
+ENV OPENSSL_FORCE_FIPS_MODE=1
+WORKDIR /go/src/app
+COPY . .
+USER 0
+RUN git log -1
+RUN make get-deps bin/mock-rbac
+
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3-1552
+LABEL idmsvc-backend=backend
+# https://developers.redhat.com/articles/2022/05/31/your-go-application-fips-compliant
+ENV OPENSSL_FORCE_FIPS_MODE=1
+RUN mkdir -p /opt/bin /opt/bin/scripts/db /opt/bin/configs
+WORKDIR /opt/bin
+COPY --from=builder /go/src/app/bin/mock-rbac ./
+COPY scripts/db/migrations /opt/bin/scripts/db/migrations
```

Comment:
Why do we need DB migrations in the mock container?

Reply:
thanks! we don't need it! "copy&paste"